libsplinter/src/service/scabbard/client/key.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+
+use sawtooth_sdk::signing::secp256k1::Secp256k1PrivateKey;
+
+use super::Error;
+
+/// Load a private key from the local filesystem.
+pub fn load_signing_key_from_file(filename: &str) -> Result<Secp256k1PrivateKey, Error> {
```

Comment:
See the discussion here - https://github.com/hyperledger/sawtooth-sabre/pull/85

Reply:
Removed from this PR